Formula/eris.rb

```diff
@@ -1,8 +1,8 @@
 class Eris < Formula
   desc "Blockchain application platform CLI"
   homepage "https://erisindustries.com"
-  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
-  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.12.0-rc1.tar.gz"
```

Comment:
This needs to be a stable (i.e. non-RC) version. You can add this version as a `devel do` block, though.